src/Elastic/Elastic.Autorest/docs/Get-AzElasticOrganizationElasticToAzureSubscriptionMapping.md

```diff
@@ -12,18 +12,11 @@ Get Elastic Organization To Azure Subscription Mapping details for the logged-in
 
 ## SYNTAX
 
-### Get (Default)
 ```
 Get-AzElasticOrganizationElasticToAzureSubscriptionMapping [-SubscriptionId <String[]>]
  [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
 ```
 
-### GetViaIdentity
```

Comment:
Why this variant is removed

Reply:
The syntax is automatically removed when the autorest tool is generated.